ui/lib/kv/addon/components/kv-subkeys-card.hbs

```diff
@@ -33,7 +33,7 @@
           <F.Label>JSON</F.Label>
         </Hds::Form::Toggle::Field>
       </div>
-      {{#if @canPatchSecret}}
```

Comment:
renamed because this no longer is just defined by capabilities, see comment below